.circleci/config.yml

```diff
@@ -78,9 +73,25 @@ workflows:
   build_and_deploy:
     jobs:
       - build
+      - aws-ecr/build-and-push-image:
+          requires:
+            - build
+          account-url: AWS_ECR_ACCOUNT_URL_ENV_VAR_NAME
+          aws-access-key-id: AWS_ACCESS_KEY_ID
+          aws-secret-access-key: AWS_SECRET_ACCESS_KEY
+          create-repo: false
+          dockerfile: Dockerfile
+          path: .
+          region: AWS_REGION
+          tag: "$(git rev-parse --abbrev-ref HEAD | sed 's/[^a-zA-Z_-]/-/g')-$(git rev-parse HEAD)"
+          repo: react-london
+          extra-build-args: "--build-arg GIT_COMMIT=$CIRCLE_SHA1"
       - deploy_to_staging:
           requires:
             - build
+            - aws-ecr/build-and-push-image
           filters:
             branches:
-              only: master
+              only:
+                - master
+                - bugfix/ci-issues
```

Comment:
did you mean to leave the bugfix branch in the filters?

Reply:
I was keeping it in, for testing but can remove it now.